tests/testing_nurbs.py

```diff
@@ -258,6 +259,34 @@ def test_nurbs_couple_nurbs_meshes(self):
         # Compare with the reference file
         compare_test_result(self, input_file.get_string(header=False))
 
+    def test_create_nurbs_base_sphere_patch_surf(self):
```

Comment:
Use the same name here as the function will have.

Reply:
I changed it to `test_nurbs_sphere_surface_reference`.